pkg/oci/repository/repository.go

```diff
@@ -73,41 +72,5 @@ func (r *Repository) Tags(ctx context.Context) ([]string, error) {
 		return nil, err
 	}
 
-	result, err = sortTags(result)
-	if err != nil {
-		return nil, err
-	}
-
-	return result, nil
-}
-
-func sortTags(tags []string) ([]string, error) {
```

Comment:
I think I would preserve at least a lexicographic order here!

Reply:
I think we should print them based on the order coming from the repository. From the most recent to the oldest one.

Reply:
Are we sure every registry will retrieve tags in that order? And even if so, imagine later in time we put a patch release of something so that you could have
latest, 0.1.0, 0.1, 0, 0.1.0-rc1, 1.0.0, 0.1.1
We can end up with mixed versions, as a user I would prefer it to be ordered some way, wdyt?

Reply:
The OCI spec says that registries must return the tags in lexical order: https://github.com/opencontainers/distribution-spec/blob/main/spec.md#listing-tags

Reply:
Alright, perfect then!

Reply:
It seems that `ghcr.io` does not comply with the spec. It returns them based on the creating order: most recent to oldest. Anyway, in the future, we could force the tags to be ordered if any use case pops out.

Reply:
Makes sense